src/Dialogs/Settings/Panels/SymbolsConfigPanel.cpp

```diff
@@ -111,6 +111,7 @@ static constexpr StaticEnumChoice trail_type_list[] = {
     N_("Vario-scaled dots with lines. "
        "Orange to red = climb. Light blue to dark blue = sink. "
        "Zero lift is presented as a yellow line.") },
+  { TrailSettings::Type::VARIO_EINK, N_("Vario E-ink"), N_("E-ink friendly color scheme, lighter and thicker dots means lift while darker and thinner means sink.") },
```

Comment:
I changed the name from VARIO_KOBO to a more generic VARIO_EINK, because I tought there are more eink devices other than Kobo where to run XCSoar (Yotaphone, to name one)